yb-voyager/src/dbzm/schemaRegistry.go

```diff
@@ -40,10 +40,14 @@ type TableSchema struct {
 	Columns []Column `json:"columns"`
 }
 
-func (ts *TableSchema) getColumnType(columnName string) (string, error) {
+func (ts *TableSchema) getColumnType(columnName string, targetDBType string) (string, error) {
 	for _, colSchema := range ts.Columns {
```

Comment:
Todo for later: we should probably store this as a map instead of a list, so that it's faster to lookup column from columnName. I suppose this code is in the critical path.

Reply:
Yes agreed!